satellite/gracefulexit/endpoint_test.go

```diff
@@ -793,17 +798,16 @@ func TestUpdatePointerFailure_DuplicatedNodeID(t *testing.T) {
 		count, ok := pieceMap[exitingNodeID]
 		require.True(t, ok)
 		require.Equal(t, 1, count)
-		count, ok = pieceMap[recNodeID]
+		count, ok = pieceMap[firstRecNodeID]
 		require.True(t, ok)
 		require.Equal(t, 1, count)
 	})
 }
 
-func testTransfers(t *testing.T, objects int, verifier func(ctx *testcontext.Context, nodeFullIDs map[storj.NodeID]*identity.FullIdentity, satellite *testplanet.SatelliteSystem, processClient exitProcessClient, exitingNode *storagenode.Peer, numPieces int)) {
-	successThreshold := 4
```

Comment:
nit - it seems weird to me for success threshold to be passed in when the min/repair thresholds used in the RSConfig to upload are kind of dependent on the value of the success threshold. I don't really have an alternative solution at the moment, but thought I'd mention that it gave me pause. I am okay approving with this change.

Reply:
I agree. Removed.